app/grants/templates/grants/card/front.html

```diff
@@ -64,6 +65,17 @@ <h2 class="grant-item__title font-subheader"><a href="{% url 'grants:details' gr
             </div>
           </div>
         </a>
+        </div>
+        {% if profile.user.is_authenticated %}
+        {% is_favorite grant profile.user as is_grant_favorite %}
+        <button class="text-decoration-none btn btn-link star-action m-0 p-0 mr-4 mt-3 gc-text-blue {% if not is_grant_favorite %}text-muted{% endif %}" data-grant="{{ grant.id }}">
+          {% if not is_grant_favorite %}
+            <i class="far fa-star" ></i><span class="ml-1 font-body">Follow</span>
+          {% else %}
+            <i class="fa fa-star" ></i><span class="ml-1 font-body">Following</span>
+          {% endif %}
+        </button>
+        {% endif %}
```

Comment:
^ indent it right
```
{% if profile.user.is_authenticated %}
    ....
{% endif %}
```

Reply:
Why are there changes in this file when this you've removed it's usage in 
https://github.com/gitcoinco/web/pull/7332/files#diff-a9ea52761a5188515132b584f9014d16L9 ?